Blog: New Categories for Matrix Spec Changes (#715)
matrix-org/matrix.org: gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx

---
date: '2020-04-29'
title: New Categories for Matrix Spec Changes
categories:
  - General
author: Andrew Morgan
image: https://matrix.org/blog/img/2020-04-28-msc-banner.png
---

On April 14th, the Spec Core Team conducted a long-overdue retrospective
about the things that were working in the
[Matrix Spec Proposal process](https://matrix.org/docs/spec/proposals),
and those that were not.

> The most glaring item on the list was the very sluggish pace that most Matrix

Reviewer: ```suggestion
The most glaring item on the list was the sluggish pace that many Matrix
```

Spec Changes (MSCs) take throughout the proposal process, as well as the
general lack of activity from the Spec Core Team members on proposals that
have not yet started a Final Comment Period.

We deeply apologize for the frustration this has likely caused many MSC
authors, and want to shed some light on the reasoning behind it, and what we
plan to do to prevent leaving authors in the dark about why there may be no
Spec Core Team activity on their proposal.

## Proposal Triaging

There are currently 136 open MSCs that have yet to undergo Final Comment
Period (FCP), 75 of which are marked as proposal-in-review, and 20 that have

> a FCP proposed. This is a lot of outstanding ideas, features and maintenance

Reviewer: We need to put this in context - how many have we merged?

changes.

The Spec Core Team itself is made up of 8 members, each of which have

> separate full-time jobs. 7 of these are employees of New Vector Ltd. All team

Reviewer: The fact that 7/8 currently work for NV is irrelevant to this, imo.

members are well-placed to be on the team given their wide breadth of
knowledge across the Matrix.org ecosystem, however the majority are some of

> the most busy pushing forward Riot, Synapse and Dendrite. This limits the

Reviewer: ```suggestion
the most busy pushing forward Matrix's reference implementations - without which, Matrix will unquestionably fail. This limits the
```

amount of MSCs that the team can effectively work on at a given time.

The team understands that there are MSCs that provide incredibly useful
features, such as

> [support for LaTeX inmessages](https://github.com/matrix-org/matrix-doc/pull/2191)

Reviewer: ```suggestion
[support for LaTeX in messages](https://github.com/matrix-org/matrix-doc/pull/2191)
```
